OptimizelySDKUniversal/generated-frameworks/Release-tvOS-universal-SDK/OptimizelySDKTVOS.framework/Headers/OPTLYProjectConfig.h

```diff
@@ -45,7 +45,7 @@ NS_ASSUME_NONNULL_END
 /// Datafile Revision number
 @property (nonatomic, strong, nonnull) NSString *revision;
 /// Flag for IP anonymization
-@property (nonatomic, assign) BOOL anonymizeIP;
+@property (nonatomic) NSNumber<Optional> *anonymizeIP;
```

Comment:
This header file is included 3 different time (iOS, TVOS, Universal).  Can't we either using soft links or have a shared framework that contained stuff like project config?

Reply:
Same response.  The original human maintained source code is in
OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.h .